services/web/client/source/class/qxapp/auth/Manager.js

```diff
@@ -26,64 +27,53 @@ qx.Class.define("qxapp.auth.Manager", {
   *****************************************************************************
   */
 
-  members:{
-    __auth: null,
-
-    setToken: function(token) {
-      // Keeps token for future requests
-      const auth = new qx.io.request.authentication.Basic(token, null);
-      this.__auth = auth;
-    },
-
-    resetToken: function() {
-      this.__auth = null;
-    },
+  members: {
 
     isLoggedIn: function() {
       // TODO: how to store this localy?? See http://www.qooxdoo.org/devel/pages/data_binding/stores.html#offline-store
       // TODO: check if expired??
       // TODO: request server if token is still valid (e.g. expired, etc)
-      return this.__auth !== null && this.__auth instanceof qx.io.request.authentication.Basic;
+      const auth = qxapp.auth.Data.getInstance().getAuth();
+      return auth !== null && auth instanceof qx.io.request.authentication.Basic;
     },
 
-    login: function(email, pass, callback, context) {
+    login: function(email, pass, successCbk, failCbk, context) {
```

Comment:
returning `qx.promise` instead of having two callbacks and a context might be nicer to work with